src/citrine/resources/sample_design_space_execution.py

```diff
@@ -0,0 +1,84 @@
+"""Resources that represent both individual and collections of sample design space executions."""
+from typing import Optional, Union, Iterator
+from uuid import UUID
+
+from citrine._rest.collection import Collection
+from citrine._session import Session
+from citrine.informatics.executions.sample_design_space_execution import SampleDesignSpaceExecution
+from citrine.informatics.design_spaces.design_space import SampleDesignSpaceInput
+from citrine.resources.response import Response
+
+
+class SampleDesignSpaceExecutionCollection(Collection["SampleDesignSpaceExecution"]):
+    """A collection of SampleDesignSpaceExecutions."""
+
+    _path_template = '/projects/{project_id}/design-spaces/{design_space_id}/sample'
+    _individual_key = None
+    _collection_key = 'response'
+    _resource = SampleDesignSpaceExecution
+
+    def __init__(self, project_id: UUID, design_space_id: UUID, session: Session):
+        self.project_id: UUID = project_id
+        self.design_space_id: UUID = design_space_id
+        self.session: Session = session
+
+    def build(self, data: dict) -> SampleDesignSpaceExecution:
+        """Build an individual SampleDesignSpaceExecution."""
+        execution = SampleDesignSpaceExecution.build(data)
+        execution._session = self.session
+        execution.project_id = self.project_id
+        execution.design_space_id = self.design_space_id
+        return execution
+
+    def trigger(
+        self, sample_design_space_input: SampleDesignSpaceInput
+    ) -> SampleDesignSpaceExecution:
+        """Trigger a sample design space execution."""
+        path = self._get_path()
+        request_dict = sample_design_space_input.dump()
+        data = self.session.post_resource(path, request_dict)
+        return self.build(data)
+
+    def register(self, model: SampleDesignSpaceExecution) -> SampleDesignSpaceExecution:
+        """Cannot register an execution."""
+        raise NotImplementedError("Cannot register a SampleDesignSpaceExecution.")
+
+    def update(self, model: SampleDesignSpaceExecution) -> SampleDesignSpaceExecution:
+        """Cannot update an execution."""
+        raise NotImplementedError("Cannot update a SampleDesignSpaceExecution.")
+
+    def list(self, *,
+             page: Optional[int] = None,
```

Comment:
`page` is deprecated, and should be removed. It probably should have been removed from everything in our last major version bump.